README.md

```diff
@@ -501,3 +501,27 @@ kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admi
 kubectl patch deploy --namespace kube-system tiller-deploy -p '{"spec":{"template":{"spec":{"serviceAccount":"tiller"}}}}'
 ```
 The above commands should resolve the errors and `helm install` should work correctly.
+
+### NFS volume mount failure: wrong fs type
```

Comment:
This change is also unrelated to this PR.

Reply:
Removed from PR.